GoogleSignInSwift/Sources/GoogleSignInButtonStyling.swift

```diff
@@ -222,6 +222,30 @@ extension GoogleSignInButtonStyle {
     case .icon: return ""
     }
   }
+
+  var minWidthForButtonText: CGFloat {
+    let bt = buttonText as NSString
+#if os(iOS) || targetEnvironment(macCatalyst)
+    let font = UIFont(name: fontNameRobotoBold, size: fontSize)
+    let rect = bt.boundingRect(
```

Comment:
It looks like the signature for boundingRect is pretty similar on macOS.  Could we dry this up?

Reply:
Yep!